aerospike-mixin/dashboards/aerospike-namespace-overview.libsonnet

```diff
@@ -0,0 +1,754 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'aerospike-namespace-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local unavailablePartitionsPanel = {
```

Comment:
these panels all look to be the same as other dashboards, so any comments applies to panels from other dashboards would apply here